application/core/QuestionTypes/ListRadio/RenderListRadio.php

```diff
@@ -58,17 +54,12 @@ public function __construct($aFieldArray, $bRenderDirect = false)
         $this->setAnsweroptions();
 
         if ($this->iNbCols > 1) {
-            // Add a class on the wrapper
             $this->sCoreClass .= " multiple-list nbcol-{$this->iNbCols}";
-            // First we calculate the width of each column
-            // Max number of column is 12 http://getbootstrap.com/css/#grid
-            $this->iColumnWidth = round(12 / $this->iNbCols);
-            $this->iColumnWidth = ($this->iColumnWidth >= 1) ? $this->iColumnWidth : 1;
-            $this->iColumnWidth = ($this->iColumnWidth <= 12) ? $this->iColumnWidth : 12;
-            $this->iMaxRowsByColumn = ceil($this->getAnswerCount() / $this->iNbCols);
```

Comment:
the good thing about this behavior was that it didnt allow the user to enter a ridiculous amount of columns, the hard limit with this is 12. Also the column-count doesnt seem to utilize the full width available.
Example with 16:
![ul2](https://user-images.githubusercontent.com/22096081/231980840-cbbbc04a-b843-475f-8e61-6f50eae704b3.PNG)
![ul3](https://user-images.githubusercontent.com/22096081/231981214-8c6407df-83c1-4297-af8a-fb2e4bcb7712.PNG)

Reply:
> the good thing about this behavior was that it didnt allow the user to enter a ridiculous amount of columns, the hard limit with this is 12.

What ? Why it's a good thing to disallow user to set 14 ? Why you want user dpon't do what they want, and if it don't work : they can review the system… If user want 14 columns : maybe there are a good reason.

> Also the column-count doesn't seem to utilize the full width available.

Columns take same height, and you set as 16 columns, i think it brother dependant and depend on the content of each choice . I don't see another way with
```
A C
B D
```
order than using column-count.
With BS : need extra element 

With 
```
A B
C D
```
we can use https://getbootstrap.com/docs/5.3/layout/grid/#row-columns
But i think it's must be a new option ([Feature request](https://bugs.limesurvey.org/view.php?id=7927))

Reply:
>  Example with 16:

Before you can not set 16 … 

Even with 8 : it don't work
![Capture d’écran du 2023-04-14 11-36-02](https://user-images.githubusercontent.com/1439428/232007723-38d5c2d3-a597-4084-98dd-eeef84774901.png)

Reply:
> Also the column-count doesnt seem to utilize the full width available.

Same with 5.X (and with same width for all part)
![Uploading Capture d’écran du 2023-04-14 11-37-03.png…]()

Reply:
> > Example with 16:
> 
> Before you can not set 16 …
> 
> Even with 8 : it don't work ![Capture d’écran du 2023-04-14 11-36-02](https://user-images.githubusercontent.com/1439428/232007723-38d5c2d3-a597-4084-98dd-eeef84774901.png)

off course it doesnt if the code code is transforming it into a equal 12 size column layout

Reply:
> 
Please stop twisting my sentences is said the "good thing" not that it is bad to allow 14. The good thing about the previous one is that it will prevent cutoff text if it limiting the size and preventing user error when playing around with css and html they dont understand.